cve_bin_tool/sbom_manager/__init__.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2021 Anthony Harrison
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-import os
+from pathlib import Path
 from collections import defaultdict
 from logging import Logger
 from typing import DefaultDict, Dict, List, Optional
```

Comment:
```suggestion
from collections import defaultdict
from logging import Logger
from pathlib import Path
from typing import DefaultDict, Dict, List, Optional
```

Reply:
isort is complaining that the imports aren't sorted correctly.  This should fix it, and then we'll be ready to merge!